aiida/orm/nodes/data/dict.py

```diff
@@ -9,6 +9,7 @@
 ###########################################################################
 """`Data` sub class to represent a dictionary."""
 import copy
+from typing import Any, Iterator, Tuple, Union
```

Comment:
The following is also useful, and often used as a standard. This way you don't have to add explicit imports while updating typing, which can be annoying
```suggestion
import typing as t
```

Reply:
I've also wondered about the least painful way of adding type hints, but now prefer to use the VScode auto-import feature. I can just type the `typing` class I want to import and Intellisense will suggest an auto-import class, which will then be automatically added here. Since this makes using the explicit imports much easier, I prefer it since the readability of the type hints is improved.